graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -35,7 +35,6 @@ def create(
         aura_ds: bool = False,
         database: Optional[str] = None,
         bookmarks: Optional[Any] = None,
-        server_version: Optional[ServerVersion] = None,
```

Comment:
why dont we pass the gds_version anymore?

Reply:
It isn't necessary. It isn't used for anything in this class.